LINK/usr/bin/miq_postgres_upgrade.sh

```diff
@@ -99,9 +104,41 @@ mount -a
 # init the new cluster
 su - postgres -c "initdb -D ${NEW_PGSQL_DIR}/data-new"
 
+sed -i.bak -e "s/^#*shared_preload_libraries.*/shared_preload_libraries = 'pglogical'/" ${NEW_PGSQL_DIR}/data-new/postgresql.conf
```

Comment:
What if pglogical is already in the list?  Also at the end of this file it says to put repmgr_funcs, but that's not listed here.  On that note, can the note at the end go away or be changed?